src/commands/apps-android-sha-create.ts

```diff
@@ -0,0 +1,56 @@
+import * as clc from "cli-color";
+import * as ora from "ora";
+
+import { Command } from "../command";
+import * as getProjectId from "../getProjectId";
+import { AppAndroidShaData, createAppAndroidSha, ShaCertificateType } from "../management/apps";
+import { requireAuth } from "../requireAuth";
+import * as logger from "../logger";
+
+async function initiateAppAndroidShaCreation(
```

Comment:
Looks like you've established a pattern here that you use in a few places, maybe you could make a new utility function like this:
```js
promiseWithSpinner(promise, message) {
  const spinner = ora(message).start();
  let data;
  try {
    data = await promise;
    spinner.succeed();
  } catch (err) {
    spinner.fail();
    throw err;
  }

  return data;
}
```

Reply:
done